---
layout: post
title: Recent Advances in Cardiac Development and Regeneration
category: English
tags: English_Posts  Heart   Development    Regeneration
keywords: Cardiac Development  Regeneration
description:  Recent Advances in Cardiac Development and Regeneration
---
                

### Abstract:                              
As the adult mammalian heart has limited potential for regeneration, the loss of cardiomyocytes during injury and disease can result in heart failure and death. The cellular processes and regulatory mechanisms involved in heart development can be exploited to repair the injured adult heart.  And pluripotent embryonic stem cells have emerged as an attractive candidate stem cell source for obtaining cardiomyocytes, because of their tremendous capacity for expansion and unquestioned potential to differentiate into cardiomyocytes. I will attempt to review the recent advances in cardiac development and regeneration from three aspects, cis-acting elements, epigenetics and regulatory networks.

                  
### Keywords: 
heart development, cardiac regeneration, cis-acting element, epigenetics, gene regulatory network.



The heart is the first organ to function in the embryo, and this function is essential for survival during fetal life. The vertebrate heart is principally composed of cardiac muscle and connective tissue. Cardiac muscle is an involuntary striated muscle tissue and is responsible for the heart's ability to pump blood. The heart grows rapidly through fetal and early postnatal life. Fetal heart growth is achieved by cardiomyocyte (CM) proliferation, but postnatal mammalian CMs lose their capacity to proliferate, and postnatal growth occurs primarily by increasing CM size rather than number. Diseases such as myocardial infarction result in the loss of billions of CMs leading to heart failure, for which we lack specific therapies. Numerous challenges must be resolved in order to achieve clinically meaningful regeneration, including the problems of producing enough cells and ensuring their functional integration. Insight into both issues comes from the study of embryonic cardiac development. In this review, I will introduce the progress of heart development and regeneration and only focus on three issues, cis-acting element activity in CM cell fate, epigenetic regulation and gene regulatory networks of the heart.    


### 1. Regulation of Heart Development by cis-Regulatory Elements
cis-Regulatory Elements (CREs) are regions of non-coding DNA, such as promoters, enhancers, insulators and silencers, which regulate the transcription of nearby or distal genes by functioning as binding sites for transcription factors [1]. Many case studies indicated that the activities of cis-acting elements and trans-acting factors can control cell differentiation and would be a major factor of cell fate determination [2-5].

The most well characterized types of CREs are enhancers and promoters and their interactions are crucial to germ cell develop to adult [6-12]. Large-scale studies have recently identified hundreds of thousands of distal enhancer elements. Emerging evidence indicates that lineage-specific transcription factors and chromatin regulators coordinate the activation of distal enhancers to ensure robust control of gene expression programs in a cell type-specific manner [13]. Transcription factor (TF) binding profiles have become a widely used proxy for genome-wide discovery of cardiac enhancers. Combinatorial binding of TFs is crucial for precisely regulating enhancer activation and the expression of nearby genes [14]. For example, recent studies have shown that the co-occupancy of cardiac TFs in the genome can identify cardiac-specific enhancers [15]. A similar investigation in Drosophila found that the collective binding of five cardiac TFs was required for full transcriptional activity of heart enhancers in vitro [16]. Using a solely computational approach, other groups identified over 40 000 putative cardiac enhancers in the human genome based on various sequence features including the co-occurrence of TF motifs [17]. Collectively, these works indicate that lineage-specific TFs act cooperatively at enhancers to drive the cardiogenic program. Certainly, enhancers and promoters are also very important to cardiac regeneration [18-20].


### 2. Epigenetic Regulation of  Cardiac Development and Regeneration
Transcriptional regulation of cardiac development has been the focus of intense investigation for several decades. The elucidation of a histone code that characterizes epigenetic control of gene expression has changed our understanding of the regulation of cell differentiation [21]. Joseph A. Wamstad et al. [22] have provided a robust global analysis of histone markers and gene expression at 4 stages of murine embryonic stem cell differentiation into cardiac myocytes. This detailed data set will provide a rich opportunity for generating and testing hypotheses related to combinatorial transcriptional regulation of gene expression and epigenetic regulation of cell fate decisions in cardiac lineages.

Cardiac DNA methylation patterns are known to correlate with gene expression and are distinct from other organs [23]. And cardiac tissue DNA methylation patterns are established during fetal developmental and during postnatal stages [24]. The role of intra-genic DNA methylation remains controversial in general [25] and has not been addressed in the heart. In contrast, the demethylation of enhancer regions has been observed in various tissues including the heart [26, 27].

Other epigenetic mechanisms including ATP-dependent chromatin remodeling, nucleosome positioning dynamics, non-coding RNA, histone acetylation and methylation can also influence heart development [28-30]. All of them are associated with adult heart regeneration and repair [31-38].


### 3. Gene Regulatory Networks in the Heart Development
Gene expression during development and differentiation is regulated in a cell- specific and stage-specific manner by complex networks of intergenic and intragenic cis-regulatory elements. Heart development is regulated by gene regulatory networks consisting of a set of highly conserved tissue-specific transcription factors, signaling molecules and non-coding RNAs [39-41]. Central to this network are the transcription factors Gata4, Mef2a, Nkx2.5, Srf, which together with their target DNA elements, form an evolutionarily conserved subcircuit essential for development [42, 43].

Recent studies have revealed central roles for microRNAs (miRNAs) as governors of gene expression during cardiac development. The integration of miRNAs into the genetic circuitry of the heart provides a rich and robust array of regulatory interactions to control cardiac gene expression [44]. For instance, Zhiyun Guo et al. [45] extracted 116 miRNAs and systematically investigated the regulatory network of transcription factor (TF)-miRNA in 12 human tissues. They identified 2347 TF-miRNA regulatory relations and revealed that most TF binding sites tend to enrich close to the transcription start site of miRNAs.  


### 4. Perspectives
If we want to overcome the fundamental problems of regenerating lost cardiomyocytes, we must fully understand the mechanism of heart development. To this end, we have to generate sufficient and detailed data about heart development, it depends on the progress of technology. So, the relevant technologies, such as the third generation sequencing and imaging techniques, are the most essential role for solving the problems. Though, of course, when we think the problems, we must consider the evolutionary origin of the heart, because “Nothing in Biology Makes Sense Except in the Light of Evolution” [46].
                                    
---                                                              
### References：                                             
[1] Patricia J. Wittkopp, Gizem Kalay. (2012). `Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence`. Nature Reviews Genetics, 13(1), 59-69.

[2] Jim R Hughes,et al. (2014). `Analysis of hundreds of cis-regulatory landscapes at high resolution in a single, high-throughput experiment`. Nature Genetics, 46(2), 205–212.

[3] Justin Cotney, et al. (2014). `The Evolution of Lineage-Specific Regulatory Activities in the Human Embryonic Limb`. Cell, 154(1), 185-196.

[4] Yin Shen, et al. (2012). `A map of the cis-regulatory sequences in the mouse genome`. Nature, 488(7409), 116–120.

[5] Diego Villar, et al. (2014). `Evolution of transcription factor binding in metazoans — mechanisms and functional implications`. Nature Reviews Genetics, 15(4), 221–233.

[6] Len A. Pennacchio,et al. (2013). `Enhancers: five essential questions`. Nature Reviews Genetics, 14(4), 288-296.

[7] Ivan Krivega, Ann Dean. (2012). `Enhancer and promoter interactions—long distance calls`. Current Opinion in Genetics & Development, 22(2), 79–85.

[8] Bing He, et al. (2014). `Global view of enhancer–promoter interactome in human cells`. PNAS, Published online May 12, 2014; doi:10.1073/pnas.1320308111.

[9] Robin Andersson, et al. (2014). `An atlas of active enhancers across human cell types and tissues`. Nature, 507(7493), 455–461.

[10] The FANTOM Consortium and the RIKEN PMI and CLST. (2014). `A promoter-level mammalian expression atlas`. Nature, 507(7493), 462–470.

[11] Wouter de Laat, et al. (2013). `Topology of mammalian developmental enhancers and their regulatory landscapes`. Nature, 502(7472), 499-506.

[12] Evgeny Z. Kvon, et al. (2014). `Genome-scale functional characterization of Drosophila developmental enhancers in vivo`. Nature, Published online 01 June 2014; doi:10.1038/nature13395.

[13] Chin-Tong Ong, et al. (2011). `Enhancer function: new insights into the regulation of tissue-specific gene expression`. Nature Reviews Genetics, 12(4), 283-93.

[14] Joseph A. Wamstad, et al. (2014). `Distal enhancers: new insights into heart development and disease`. Trends in cell biology, 24(5), 294-302.

[15] Aibin He, etal. (2011). `Co-occupancy by multiple cardiac transcription factors identifies transcriptional enhancers active in heart`. PNAS, 108(14), 5632-5637.

[16] Guillaume Junion, et al. (2012). `A Transcription Factor Collective Defines Cardiac Cell Fate and Reflects Lineage History`. Cell, 148(3), 473-486.

[17] Leelavati Narlikar, et al. (2010). `Genome-wide discovery of human heart enhancers`. Genome Research, 20(3), 381-392.

[18] Li Qian, et al. (2012). `In vivo reprogramming of murine cardiac fibroblasts into induced cardiomyocytes`. Nature, 485(7400), 593–598.

[19] Kunhua Song, et al. (2012). `Heart repair by reprogramming non-myocytes with cardiac transcription factors`. Nature, 485(7400), 599–604.

[20]  Guo N. Huang, et al. (2012). `C/EBP Transcription Factors Mediate Epicardial Activation During Heart Development and Injury`. Science, 338(6114), 1599-603.

[21] Michael S. Parmacek, Jonathan A. Epstein. (2013). `An Epigenetic Roadmap for Cardiomyocyte Differentiation`. Circulation Research, 112(6), 881-883.

[22] Joseph A. Wamstad, et al. (2012). `Dynamic and coordinated epigenetic regulation of developmental transitions in the cardiac lineage`. Cell, 151(1), 206–220.

[23] Xie L, et al. (2011). `An integrative analysis of DNA methylation and RNA-Seq data for human heart, kidney and liver`. BMC Systems Biology, 5 (Suppl 3):S4.

[24] Liang P, et al. (2011). `Genome-wide survey reveals dynamic widespread tissue-specific changes in DNA methylation during development`. BMC Genomics, 12, 231.

[25] Jones PA. (2012).` Functions of DNA methylation: islands, start sites, gene bodies and beyond`. Nature Review Genetics, 13:484–492.

[26] Hon GC, et al. (2013). `Epigenetic memory at embryonic enhancers identified in DNA methylation maps from adult mouse tissues`. Nature Genetics, 45,1198–1206.

[27]  Michael J. Ziller, et al. (2013). `Charting a dynamic DNA methylation landscape of the human genome`. Nature, 500, 477–481.

[28] Thomas Nührenberg, et al. (2014). `Epigenetics in cardiac development, function, and disease`. Cell Tissue Research, 356(3), 585-600.

[29] Enzo R. PORRELLO. (2013). `microRNAs in cardiac development and regeneration`. Clinical Science, 125(4), 151–166.

[30] Carla A. Klattenhoff, et al. (2013). `Braveheart, a Long Noncoding RNA Required for Cardiovascular Lineage Commitment`. Cell, 152(3), 570–583.

[31] Mei Xin, et al. (2013). `Mending broken hearts: cardiac development as a basis for adult heart regeneration and repair`. Nature Reviews Molecular Cell Biology, 14(8), 529-541.

[32] Harsha Pawani, Deepa Bhartiya. (2013). `Pluripotent stem cells for cardiac regeneration: overview of recent advances & emerging trends`.   Indian Journal of Medical Research, 137(2), 270-282.

[33] Zhiqiang Lin, William T. Pu. (2014). Strategies for Cardiac Regeneration and Repair. Science Translational Medicine, 6(239), 239rv1.

[34] James J. H. Chong, et al. (2014). `Human embryonic-stem-cell derived cardiomyocytes regenerate non-human primate hearts`. Nature, 510(7504), 273-277.

[35] Xiulan Yang, et al. (2014). `Maturation of Human Pluripotent Stem Cell–Derived Cardiomyocytes`.  Circulation Research, 114(3),  511-523.

[36] Mark Mercola, et al. (2011).  `Cardiac muscle regeneration: lessons from development`. Genes & Development, 25(4), 299–309.

[37] Vincent M. Christoffels, William T. Pu. (2013). `Developing insights into cardiac regeneration`. Development, 140(19), 3933-3937.

[38] Ruilin Zhang, et al. (2013). `In vivo cardiac reprogramming contributes to zebrafish heart regeneration`. Nature, 498(7455), 497–501.

[39] Eric N. Olson. (2006). `Gene regulatory networks in the evolution and development of the heart`. Science, 313(5795), 1922-1927.

[40] Benoit G. Bruneau. (2013). `Signaling and transcriptional networks in heart development and regeneration`. Cold Spring Harbor Perspectives in Biology, 5(3), a008292.

[41] Nikhil V. Munshi. (2012). `Gene Regulatory Networks in Cardiac Conduction System Development`. Circulation Research, 110(11), 1525-37.

[42] Hailin Che, et al. (2014). `A provisional gene regulatory atlas for mouse heart development`. PLoS One, 9(1), e83364.

[43] Jenny Schlesinger, et al. (2011). `The cardiac transcription network modulated by Gata4, Mef2a, Nkx2.5, Srf, histone modifications, and microRNAs`. PLoS Genetics, 7(2), e1001313.

[44] Zhiyun Guo, et al. (2014). `Genome-wide survey of tissue-specific microRNA and transcription factor regulatory networks in 12 tissues`. Scientific Reports, 4, 5150.

[45] Ning Liu, Eric N. Olsonemai. (2010). `MicroRNA Regulatory Networks in Cardiovascular Development`. Cell, 18(4), 510–525.

[46] Theodosius Dobzhansky. (1973). `Nothing in Biology Makes Sense Except in the Light of Evolution`. The American Biology Teacher, 35, 125-129.


这是我以前的生理学课程期末论文， 本来想把 ncRNA regulation, Nonlinear and stochastic dynamics model and Machine Learning Model in the heart development, Tissue Engineering and Synthetic Biology in cardiac regeneration, 都加进去， 但由于时间不够了， 老师也说过五六页就够了，所以就省掉了。                                                                                        
