---
layout: post
title: Relationship between Histone Modifications and Nucleosome Turnover
category: ReadingNotes
tags: Histone_Modifications   Nucleosome_Turnover
keywords: Histone Modifications, Nucleosome Turnover
description:  Relationship between Histone Modifications and Nucleosome Turnover
---

Apparently, all the factors that influence nucleosome turnover can be classified into two categories:  internal and external causes.  The external causes mainly contain chromatin remodeling complexes, some enzymes, histone chaperones and DNA binding proteins [1,2], such as ATP-dependent chromatin assembly and remodelling factor (ACF) [3]. The internal causes contain all the factors that influence the histone-DNA or histone-histone interactions, such as DNA flexibility, DNA methylation, histone modification and variants [4-7]. Here, I only introduce the relationship between histone modifications and nucleosome turnover.


Recent data suggest that histone modifications have a direct effect on nucleosomal architecture by affecting histone–histone and histone–DNA interactions, as well as the binding of histones to chaperones [8]. And Venkatesh [9, 10] show that co-transcriptional acetylation is achieved in part by histone exchange over open reading frames (ORFs) in yeast.


Firstly, H3K56ac is positioned at the entry–exit point of the DNA on the nucleosome, it can form water-mediated contacts between the histone residue and the DNA. So H3K56ac might influence nucleosome stability and DNA accessibility by enabling the nucleosome breathing. Arg42 is another residue in histone H3 that is located at the DNA entry–exit region of the nucleosome. The addition of a methyl group to an Arg residue not only adds steric bulk but also removes a potential hydrogen bond donor, which suggests that this modification affects the interaction of histones with the DNA. And previous data imply a role for asymmetric dimethylation of H3R42 in decreasing nucleosome stability.


Secondly, H3K122 is located on the dyad axis of the nucleosome, in which the interaction between histones and DNA is strongest. H3K122ac can establish a water mediated salt bridge between H3K122 and the DNA. Previous data suggest that fully acetylated H3K64 are less stable than nucleosomes containing unmodified H3K64. In addition, and H3K64 acetylation reduces the interaction with DNA.


Thirdly, H3K79 is located on the outside of the nucleosome, in the solvent-exposed carboxy-terminal end of the H3 α1 helix. The structure of chemically dimethylated H3K79 showed that this modification does not cause a major change in nucleosome structure, but a subtle reorientation of the region surrounding Lys79, which probably results in the loss of a single hydrogen bond to the L2 loop of H4. And H4K91 is positioned within the H3–H4 tetramer–H2A–H2B dimer interface and mutation mimicking the acetylated state leads to decondensed chromatin and loss of nucleosomal interaction.


Fourthly, histone chaperones are acidic proteins and hence ideally suited to guide the basic histones to their final destination within chromatin. They either bind the histone–histone interface or part of the histone–DNA interaction surface. Therefore, histone modifications, such as H3K56ac and H2AQ105, could also modulate the binding to their respective chaperones.


Lastly, in yeast, co-transcriptional acetylation is achieved in part by histone exchange over ORFs. In addition to its function of targeting and activating the Rpd3S complex, H3K36 methylation suppresses the interaction of H3 with histone chaperones, histone exchange over coding regions and the incorporation of new acetylated histones.


In a words, histone modifications and nucleosome dynamics can influence each other. So understanding the correlation between histone modification and nucleosome turnover maybe become the first task.




###References:                       
[1] Zachary A. Gurard-Levin, et al. (2014). Histone Chaperones: Assisting Histone Traffic and Nucleosome Dynamics. Annual Review of Biochemistry, 83: 487-517.                    
[2] Maria J.E. Koster, et al. (2015). Genesis of Chromatin and Transcription Dynamics in the Origin of Species. Cell, 161(4), 724–736.                       
[3] Timothy R. Blosser, et al. (2009). Dynamics of nucleosome remodelling by individual ACF complexes. Nature, 462, 1022-1027.                
[4] Isabel Jimenez-Useche, et al. (2013). DNA Methylation Regulated Nucleosome Dynamics. Scientific Reports, 3, Article number: 2121.                   
[5] Thuy T.M. Ngo, et al. (2015).  Asymmetric Unwrapping of Nucleosomes under Tension Directed by DNA Local Flexibility. Cell, 160(6), 1135–1144.                          
[6] Gregory D. Bowman, Michael G. Poirier. (2015). Post-Translational Modifications of Histones That Influence Nucleosome Dynamics. Chem. Rev., 115 (6), 2274–2295.                      
[7] Hideaki Tagami, et al. (2004). Histone H3.1 and H3.3 Complexes Mediate Nucleosome Assembly Pathways Dependent or Independent of DNA Synthesis. Cell, 116(1), 51–61.                              
[8] Peter Tessarz, Tony Kouzarides. (2014). Histone core modifications regulating nucleosome structure and dynamics. Nature Reviews Molecular Cell Biology, 15, 703–708.                       
[9] Swaminathan Venkatesh, et al. (2012). Set2 methylation of histone H3 lysine 36 suppresses histone exchange on transcribed genes. Nature, 489, 452–455.                    
[10] Michaela Smolle, et al. (2012). Chromatin remodelers Isw1 and Chd1 maintain chromatin structure during transcription by preventing histone exchange. Nature Structural & Molecular Biology, 19, 884–892.                        
                                                               
